packages/components/inputs/rich-text-utils/src/index.ts

```diff
@@ -2,6 +2,7 @@ export * as localized from './localized';
 export { default as html } from './html';
 export { default as isEmpty } from './is-empty';
 export { default as richTextPlugins } from './rich-text-plugins';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
I don't think we are supposed to have this eslint comment

Reply:
I think the reason you're getting the error is because of the file name. It is `rich-text-body/index.Ts` instead of `rich-text-body/index.ts`

Reply:
Ah! thank you. Nice catch.